api/src/routers/all-meals.js

```diff
@@ -0,0 +1,10 @@
+import express from "express";
+import knex from "../database_client.js";
+
+
+const allMeals = express.Router();
+allMeals.get('/', async(req , res)=> {
+    const all_meals = await knex('meal').orderBy('id')
```

Comment:
You're mixing snake_case and camel Case. In JavaScript, we normally use camel Case

https://developer.mozilla.org/en-US/docs/Glossary/Camel_case